src/Aspire.ProjectTemplates/templates/aspire-empty/AspireApplication.1.ServiceDefaults/Extensions.cs

```diff
@@ -31,10 +32,16 @@ public static IHostApplicationBuilder AddServiceDefaults(this IHostApplicationBu
             http.AddServiceDiscovery();
         });
 
+        // Uncomment the following to restrict the allowed schemes for service discovery.
+        // builder.Services.Configure<ServiceDiscoveryOptions>(options =>
+        // {
+        //     options.AllowedSchemes = ["https"];
+        // });
+
```

Comment:
Hmm where did this come from? I'm not seeing it in the current templates. Did we have this and then remove it at some point?

Reply:
Ah got it, thanks. It's fine to add it to the other templates then, but we don't need it in the playground apps.

Reply:
OK, it's now removed from the playground apps